frontend/packages/dev-console/integration-tests/features/helm/install-HelmChart.feature

```diff
@@ -12,12 +12,12 @@ Feature: Install the Helm Release
         Then user can see "Helm Chart" card on the Add page
 
 
-    @regression, @smoke, @manual
+    @regression, @smoke
```

Comment:
As discussed, please remove the @regression tag when both [@regression, @smoke,] are on scenarios

Reply:
done, 73c363ec6787648fc93c8143b395c02f13b28abc